code/WorkInProgress/pomf/spacepods/spacepods.dm

```diff
@@ -393,9 +393,14 @@
 		return
 	if(!Adjacent(usr) || !Adjacent(over))
 		return
-	if(!occupants.Find(usr))
-		return ..() //Handle mousedrop T
 	var/turf/T = get_turf(over)
+	if(!occupants.Find(usr))
+		visible_message("<span class='notice'>[usr] start pulling [get_pilot().name] out of \the [src].</span>")
+		if(do_after(usr, src, 4 SECONDS))
+			move_outside(get_pilot(), T)
+			add_fingerprint(usr)
+		return
+		//return ..() //Handle mousedrop T
```

Comment:
This comment is useless and confusing, remove it.

Reply:
done